whois-api/src/test/java/net/ripe/db/whois/api/fulltextsearch/FullTextSearchNoIndexTestIntegration.java

```diff
@@ -2,19 +2,19 @@
 
 import net.ripe.db.whois.api.AbstractIntegrationTest;
 import net.ripe.db.whois.api.RestTest;
-import net.ripe.db.whois.common.IntegrationTest;
-import org.junit.BeforeClass;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
+
+import org.junit.jupiter.api.BeforeAll;
+
+import org.junit.jupiter.api.Test;
 
 import javax.ws.rs.NotFoundException;
 
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.fail;
 
-@Category(IntegrationTest.class)
+@org.junit.jupiter.api.Tag("IntegrationTest")
```

Comment:
Are the Tag annotation inherited? Can you add the annotation to the ```AbstractIntegrationTest``` only?

Reply:
I tried that but it does not work. seems like Tags are not inherited

Reply:
Confirmed that the Tags class uses the ```Inherited``` annotation, but the inheritance only applies from superclasses, not from interfaces, see the Inherited class JavaDoc:
```
Note also that this meta-annotation only causes annotations to be inherited from superclasses; annotations on implemented interfaces have no effect.
```